elfi/methods/bo/acquisition.py

```diff
@@ -327,16 +327,20 @@ class MaxVar(AcquisitionBase):
 
     """
 
-    def __init__(self, quantile_eps=.01, *args, **opts):
+    def __init__(self, model, prior, quantile_eps=.01, **opts):
```

Comment:
i actually interpreted `**opts` as the optional and `*args` as the required arguments from the parent class, and this confusion is essentially why i removed them. i generally think that it’s better to repeat the required arguments so that one doesn’t have to check the parent class, but the more important reason in this case was that the previous init signature was difficult to understand. we can see from the signature that `quantile_eps` is optional and from the base class that the only required (positional) argument is `model`. however while `acq = LCBSC(model)` works,  `acq = MaxVar(model)` does not. one needs to use either `acq  = MaxVar(model=model)` or `acq = MaxVar(0.01, model)`, and while this may be obvious to an experienced person, it can be a bit confusing to someone with less experience.

the above information was also incorrect since `MaxVar` also needs `prior`. in other words `acq = MaxVar(model=model)` that was initialised without `prior` would actually crash when we run BOLFI. i interpreted this so that `prior` should be a required argument, but then worried that updating to `MaxVar(prior, quantile_eps=0.01, *args, **opts)` would be even more confusing than the current version. and hence the update :)